src/create/SetRating.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react'
+import { setLocal } from "../services";
+import {RatingCategoryStyle, Rating, RatingIcon} from '../cards/CafeCardStyles'
+import {AllRatings, RatingLabel} from './FormStyles'
+import WorkIcon from '../images/WorkAtmosphereY.png'
+import WorkIconDisabled from '../images/WorkAtmosphereCreate.png'
+import WlanIcon from '../images/WLANY.png'
+import WlanIconDisabled from '../images/WLANCreate.png'
+import CupIcon from '../images/CupCreateY.png'
+import CupIconDisabled from '../images/CupCreateN.png'
+
+export default function SetRating() {
+
+    const [activeWorkIndex, setActiveWorkIndex] = useState(-1)
```

Comment:
-1 means undefined --> Is that what you would like as a start value in your state?